src/main/java/tds/exam/results/messaging/ExamCompletedMessageListener.java

```diff
@@ -37,6 +44,10 @@ public void handleMessage(final String examId) {
         LOG.debug("Received completed exam notification for id: {}", examId);
         final TDSReport report = examResultsService.findExamResults(UUID.fromString(examId));
 
-        //TODO ship the report somewhere
+        try {
+            testIntegrationSystemService.sendResults(UUID.fromString(examId), report);
```

Comment:
I guess I would like this listener to take an exam id and hand it off to a service that will do all the processing.  This listener should simply know how to read a queue and respond to the message on that queue.